styles.css

```diff
@@ -458,3 +460,399 @@ form {
+    margin-right: 0;
+    height: 3vh;
+    margin-top: 0;
+    margin-bottom: 0;
+    align-self: center;
+  }
+
+  .about-project {
+    margin: 2%;
+  }
+
+  .buttons {
+    margin: 2%;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .see-project {
+    margin-left: 7%;
+    padding: 0 1% 0 1%;
+    font-size: 1.3vw;
+    width: 10vw;
+    align-self: start;
+  }
+
+  .contact {
+    width: 50%;
+  }
+
+  .logo,
+  .toolbar-links > a,
+  .about,
+  h2,
+  .about-project {
+    font-size: 1.3vw;
+    line-height: 1.7;
+  }
+
+  .greeting {
+    font-size: 3vw;
+    grid-row: 2/3;
+  }
+
+  .second-text {
+    grid-row: 4/6;
+  }
+
+  .container > ul > li {
+    font-size: 1.2vw;
+    font-weight: 50;
+    margin: 0;
+  }
+
+  .buttons > ul > li {
+    margin: 2%;
+    font-size: 0.8vw;
+    font-weight: 50;
+    padding: 0.07rem 0.3rem 0.07rem 0.3rem;
+    border-radius: 0.3rem;
+  }
+
+  .buttons > ul {
+    margin: 0;
+  }
+
+  .container > ul > .name {
+    font-size: 1.1vw;
+  }
+
+  .dropdown-content > a > img {
+    padding: 0;
+    width: 4.5vw;
+    grid-row: 1/2;
+    grid-column: 1/2;
+  }
+
+  .socials > ul > li > a > img {
+    width: 1.5vw;
+  }
+
+  /* about myself */
+  #headline-two {
+    display: grid;
+    grid-template-columns: repeat(2, 40% 60%);
+    grid-template-rows: repeat(9, 5vh);
+    height: 55vh;
+    padding: 8.2% 8.2% 4.2% 8.2%;
+    background-color: #fff;
+    border-radius: 0 20% 0 0;
+  }
+
+  .wrapper-contact > div > h2 {
+    font-size: 3vw;
+  }
+
+  #headline-two > h2 {
+    grid-row: 8/9;
+  }
+
+  #headline-two > .socials {
+    grid-row: 9/10;
+    align-self: center;
+  }
+
+  #about-myself {
+    height: 60vh;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-image: none;
+    background-color: #f7f7f9;
+  }
+
+  #resume {
+    grid-row: 10/11;
+    width: 13vw;
+    padding: 0;
+  }
+
+  .dropbtn {
+    font-size: 1.4vw;
+    padding: 1rem;
+  }
+
+  .dropdowns {
+    grid-column: 2 / 3;
+    grid-row: 2/6;
+  }
+
+  .list-of-dropdowns {
+    margin: 0;
+  }
+
+  .dropdown-content {
+    display: none;
+  }
+
+  .list-of-dropdowns > li:hover .dropdown-content {
+    display: flex;
+    flex-direction: row;
+    justify-content: start;
+  }
+
+  .dropdown-content > a {
+    font-size: 1.3vw;
+    height: 7.5vw;
+    width: 7.5vw;
+    display: grid;
+    grid-template-rows: 70% 30%;
+    padding: 2%;
+  }
+
+  .dropdown-content > a > span {
+    grid-row: 2/3;
+    grid-column: 1/3;
+    font-weight: 250;
+    color: #253858;
+  }
+
+  /* contact form */
+
+  #contact-form {
+    height: 65vh;
+    margin: 0;
+    padding: 0;
+    background-color: #fff;
+    margin-top: 10%;
+  }
+
+  .wrapper-contact {
+    background-image: url(images/desktop-pictures/contact\ form\ background\ shapes.png);
+    background-size: contain;
+    border-radius: 20% 0 0 0;
+    padding: 3%;
+    margin: 0;
+    height: 80vh;
+  }
```

Comment:
- [ ] It keeps getting closer but the background image in the contact section seems to be a bit misaligned like the previous landing page so please make sure to positon it properly so it looks similar to the Figma design:
- Figma design:
  ![image](https://github.com/agneta1991/Portfolio/assets/104968236/b7444731-b9d8-426b-829c-75b417273aab)

- Your design:
  ![image](https://github.com/agneta1991/Portfolio/assets/104968236/e71eeeaf-7b66-4d6a-b5c7-492a4139c8ff)